plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraClientConfig.java

```diff
@@ -380,13 +380,14 @@ public CassandraClientConfig setNoHostAvailableRetryTimeout(Duration noHostAvail
     }
 
     @Min(1)
-    public int getSpeculativeExecutionLimit()
+    @Nullable
+    public Integer getSpeculativeExecutionLimit()
```

Comment:
nit: `Optional<Integer>`?

Reply:
Changed to `Optional<@Min(1) Integer>` as we can't use `@Min(1) public Optional<Integer>`.